presto-parser/src/main/java/io/prestosql/sql/SqlFormatter.java

```diff
@@ -393,7 +394,21 @@ protected Void visitSingleColumn(SingleColumn node, Integer indent)
         @Override
         protected Void visitAllColumns(AllColumns node, Integer context)
         {
-            builder.append(node.toString());
+            if (node.getTarget().isPresent()) {
```

Comment:
This could simplify to
```java
node.getTarget().ifPresent(value -> builder
        .append(formatExpression(value, parameters))
        .append("."));
builder.append("*");